Server/Data/generateData.js

```diff
@@ -0,0 +1,41 @@
+const faker = require('faker');
+let priceModel = require('./db.js')
+
+const seedDataBase = () => {
+  const getColors = () => {
+    let colors = [];
+    for(let c = 0; c < 5; c++) {
+      colors.push(faker.commerce.color())
+    }
+    return colors;
+  }
+
+  let recordsContainer = [];
+
+    console.log('Seeding DB')
+    for(let i = 0; i < 100; i++) {
+      let itemColors =  getColors();
+      let newPriceRecord =  new priceModel(
```

Comment:
Seems like extraneous to create new model here for each record. Can create an object with the properties shown here and push to your records container. Can then use the same insertMany method